### ISSUE TITLE
NVDA doesn't read the change in slide when I go to previous and next slides in Libre Office Impress

### ISSUE THREAD

**Issue by Bibeksharma5** [nvaccess/nvda#20066]
### Brief summary

Hello NVAccess team, when I press Page up/page down key to jump between previous and next page, I'm able to move between slides (in slide editing view). Ctrl+Home and Ctrl+End are also working for navigating to start and end of the slide. However, NVDA is not speaking "Slide 1", "slide 2" or others. NVDA should indicate when I move between slides. So, please fix this. Regards.

### Steps to reproduce

1. I opened Libre Office.
2. Under recents, I pick a pptx file and press enter to open.
3. I tried to move between slides which I explained above with Page up, Page down, Ctrl+Home and Ctrl+End. But, NVDA is not indicating the slide movement.

### Actual behavior

NVDA is not indicating the movement of slides when I move between them.

### Expected behavior

While I'm navigating between different slides, NVDA should tell "slide 1, slide 2, slide 5, slide 8" and others which is self explanatory.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/27437363/nvda.log)
[nvda-old.log](https://github.com/user-attachments/files/27437362/nvda-old.log)

### NVDA type

installed copy

### NVDA version

2026.1rc1 (2026.1.0.55678)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

No.

### Windows version

Windows 11 25H2 (AMD64) build 26200.8246

### Name and version of other software in use when reproducing the issue

Libre office Version: 25.8.5.2 (X86_64)

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have run the System Accessibility Repair Tool and the issue still occurs

**Comment by XLTechie** [comment]
CC @michaelweghorn

**Comment by Bibeksharma5** [comment]
What do you mean?

**Comment by XLTechie** [comment]
I was asking a particular person to look at this issue.

**Comment by Bibeksharma5** [comment]
OK! Sorry.
